packages/core/codeframe/test/codeframe.test.js

```diff
@@ -752,4 +754,46 @@ describe('codeframe', () => {
     assert.equal(lines[2], '> 2 | ew line new line ne');
     assert.equal(lines[3], '>   | ^^^^^^ I have a message');
   });
+
+  it('Should pad properly, T-650', () => {
+    let fileContent = readFileSync(
+      joinPath(__dirname, './fixtures/wordpress-components.js'),
+      'utf8',
+    );
+    let codeframeString = codeframe(
+      fileContent,
+      [
+        {
+          start: {
+            line: 8,
+            column: 10,
+          },
+          end: {
+            line: 8,
+            column: 48,
+          },
+        },
+      ],
+      {
+        useColor: false,
+        syntaxHighlighting: false,
+        language: 'js',
+        terminalWidth: 100,
+      },
+    );
+
+    let lines = codeframeString.split(LINE_END);
+    assert.equal(lines.length, 5);
+    assert.equal(lines[0], '  7  |  */');
+    assert.equal(
+      lines[1],
+      `> 8  | import { unstable_CompositeItem as CompositeItem } from 'reakit/Composite';`,
+    );
+    assert.equal(
+      lines[2],
+      '>    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^',
+    );
+    assert.equal(lines[3], '  9  | /**');
+    assert.equal(lines[4], '  10 |  * Internal dependencies');
```

Comment:
Line numbers are usually right-aligned:
```
 9 | /**
10 |  * Internal dependencies
```

Reply:
you're right... it should be a padStart

Will update it, shouldn't be that much work